src/components/header.js

```diff
@@ -5,12 +5,13 @@ import React from "react"
 const Header = ({ siteTitle }) => (
   <header
     style={{
-      background: `rebeccapurple`,
-      marginBottom: `1.45rem`,
+      background: `#f8cd2f`,
+      // marginBottom: `1.45rem`,  //commented out so it doesn't mess up the About component
```

Comment:
Same thing with this styling. When we're mobbing, we'll either delete this or uncomment it - whatever makes the most sense. (It's ok to change code that's been worked on by other people as we all work on this together.)

Reply:
Removed entire `marginBottom` line